components/user/form/PasswordInput.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <v-responsive max-width="400">
+    <VTextField
+      variant="outlined"
+      clearable
+      :rules="[rules.required, rules.min]"
+      :append-inner-icon="ShowPassord ? 'mdi-eye' : 'mdi-eye-off'"
+      :type="ShowPassord ? 'text' : 'password'"
+      :label="confirmPassword ? 'Bekreft passord' : 'Passord'"
```

Comment:
I did not quite understand this one.

Reply:
> I did not quite understand this one.

As the text _Bekreft passord_ and _Passord_ is hardcoded into the component label, this will not be intuitive if for example the user has toggled the english version of the website. Hence, you can instead use a `$t()` like in the example below to reference a spesific key in the language JSON files underneath `locales/`. To fix this, try to change
```
:label="confirmPassword ? 'Bekreft passord' : 'Passord'"
```
to
```
:label="confirmPassword ? $t('user.register.password_confirmation') : $t('user.register.password')"
```

Reply:
Should then also add the keys, and text to the keys, in English and Norwegian in locales/?